docs/what-is-feldera.md

```diff
@@ -4,18 +4,17 @@ slug: /
 
 # What is Feldera?
 
-Feldera **Continuous Analytics Platform**, also known as
-**Feldera Platform**, is a set of containerized services packaging a fast
-data computational engine for **continuous analytics**. Feldera Platform allows
+Feldera is a set of containerized services packaging a fast
+data computational engine for **continuous analytics**. Feldera allows
 users to run continuous queries directly on data in motion, without storing
 the data in databases or storage systems, before querying.
 
-With Feldera Platform, DML and DDL SQL written for popular databases can operate
+With Feldera, DML and DDL SQL written for popular databases can operate
```

Comment:
We don't have DML do we? @ryzhyk

Reply:
we don't

Reply:
The compiler does, and that's how it's tested. And that can be the basis of a cli

Reply:
Let's drop the DML mention. If it's not usable from the offered product, it shouldn't be in the documentation.

Reply:
OK, I changed "DML and DDL SQL" to just "SQL".